stream_alert_cli/test.py

```diff
@@ -53,6 +53,33 @@ def report_output(cols, failed):
 
     print '\t{}\ttest ({}): {}'.format(status, *cols)
 
+def report_output_summary(rules_fail_pass):
+    """Helper function to print the summary results of all tests
+    Args:
+        rules_fail_pass [list]: A list containing two lists for failed and passed
+            rule tests. The sublists contain tuples made up of: (rule_name, rule_description)
+    """
+    failed_tests = len(rules_fail_pass[0])
```

Comment:
hm that would be cool but kinda hard to catch... errors are just printed along the way and don't bubble up. any suggestions?